Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM alpine:3.16
+
+# set some defaults
+ENV AWS_DEFAULT_REGION "us-east-1"
+ENV KUBECTL_VER=v0.23.10
+# https://github.com/sgerrand/alpine-pkg-glibc/releases
+ENV GLIBC_VER=2.33-r0
```

Comment:
I don't think you should install glibc. Alpine can behave oddly when you don't just use musl. 

We do that here:
https://bitbucket.org/ontsys/dind-rootless-awscli/src/5e0495bddfb7cff7eb507cace3643986baa4b7b1/Dockerfile#lines-3:30,47:51